VAMobile/src/api/claimsAndAppeals/getClaimsAndAppeals.tsx

```diff
@@ -45,12 +45,13 @@ export const useClaimsAndAppeals = (claimType: ClaimType, options?: { enabled?:
   const claimsInDowntime = useDowntime(DowntimeFeatureTypeConstants.claims)
   const appealsInDowntime = useDowntime(DowntimeFeatureTypeConstants.appeals)
 
+  const claimsAndAppealsUp = !claimsInDowntime && !appealsInDowntime
   const queryEnabled = options && has(options, 'enabled') ? options.enabled : true
-  const claimsAndAppealAccess = authorizedServices?.claims || authorizedServices?.appeals
+  const claimsAndAppealAccess = authorizedServices?.claims && authorizedServices?.appeals
```

Comment:
Wont this lock users out from both claims and appeals if they're missing the other permission?